src/main/java/meteordevelopment/meteorclient/systems/modules/misc/Spam.java

```diff
@@ -76,6 +76,15 @@ public class Spam extends Module {
         .build()
     );
 
+    // PMspam button
+    private final Setting<Boolean> PMspam = sgGeneral.add(new BoolSetting.Builder()
+        .name("PMspam")
+        .description("Parses playersList and /msg them")
```

Comment:
Avoid using developer syntax in descriptions, most of Meteor Client's userbase aren't used to it. You should just say "the player list" instead of "playersList".